helpers/sony-bravia-android-tv-communicator.js

```diff
@@ -1,6 +1,6 @@
 const Homey = require('homey');
 
-const Fetch = require('node-fetch');
+const Fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args));

```

Comment:
Why do we need this dynamic import?

Reply:
Incase you update node-fetch to a newer version such as ^3.3.0, eslint will fail.

Reply:
Oh, actually I'm getting a warning of the current version used, that the are 2 vulnerabilities in it.